src/main/java/resources/PersonsResource.java

```diff
@@ -59,12 +53,20 @@ public Persons createPerson(Persons person) {
     @GET
     @Path("/all")
     @UnitOfWork
-    public List fetchPersonsWithSlots(@QueryParam("startDate") String startDate, @QueryParam("endDate") String endDate) {
+    public List fetchPersonsWithSlots(@QueryParam("startDate") String startDate, @QueryParam("endDate") String endDate) throws ParseException {
         Date start = DateTime.parse(startDate).toDate();
         Date end = DateTime.parse(endDate).toDate();
 
         List<Persons> persons = personsDao.findAll();
         persons.forEach(p -> p.setSlotsList(slotsDao.getForPersonForWeek(p.getId(), start, end)));
+        persons.forEach(p -> {
```

Comment:
I don't like this coupling of notes with person - we should retrieve them separately, in different requests. This should be added to issues and changed later